uaclient/snap.py

```diff
@@ -0,0 +1,87 @@
+from typing import List, Optional
+
+from uaclient import apt, util
+
+SNAP_CMD = "/usr/bin/snap"
+SNAP_INSTALL_RETRIES = [0.5, 1.0, 5.0]
+HTTP_PROXY_OPTION = "proxy.http"
+HTTPS_PROXY_OPTION = "proxy.https"
+
+
+def is_installed() -> bool:
+    """Returns whether or not snap is installed"""
+    return "snapd" in apt.get_installed_packages()
```

Comment:
While this test is "valid" can we more simply check util.which(SNAP_CMD)  which we do when we "enable" livepatch anyway?